src/UI/UIUtils.java

```diff
@@ -126,15 +126,19 @@ private static void initCosmetics()
           else if(fileEntry.canRead() && !fileEntry.getName().endsWith(DEFAULT_FACTION_NAME)) // However, we don't wanna add our default twice
           {
             String basis = DEFAULT_FACTION_NAME;
-            try
+            String basisPath = fileEntry.getAbsolutePath() + "/basis.txt";
+            if( new File(basisPath).canRead() )
             {
-              BufferedReader br = new BufferedReader(new FileReader(fileEntry.getAbsolutePath() + "/basis.txt"));
-              basis = br.readLine().trim();
-              br.close();
-            }
-            catch (IOException ioex)
-            {
-              // if there's no valid basis, there's no problem
+              try
+              {
+                BufferedReader br = new BufferedReader(new FileReader(basisPath));
+                basis = br.readLine().trim();
+                br.close();
+              }
+              catch (IOException ioex)
+              {
+                System.out.println("WARNING! Exception loading faction basis " + basisPath);
```

Comment:
Whenever I print a scary warning like this that doesn't stop execution, I like to print what I'm doing instead that allows me to not halt execution. In this case probably just something like `WARNING! Exception loading faction sprite basis! Defaulting to Thorn sprites`. That way the user knows how much they need to care about the warning or if they should try to fix it.